crates/holochain/src/conductor/conductor.rs

```diff
@@ -2816,57 +2819,51 @@ impl Conductor {
         }
 
         // Find any DNAs from cleaned up cells which don't have representation in any cells
-        // in any app. In other words, find the DNAs which are *only* represented in uninstalled apps.
+        // in any installed app, so that we can remove their data from the databases.
         let all_dnas: HashSet<_> = all_cells
             .into_iter()
             .map(|cell_id| cell_id.dna_hash())
             .collect();
-        let dnas_to_cleanup = cells_to_cleanup
+        let dnas_to_purge = cells_to_cleanup
             .iter()
             .map(|cell| cell.id().dna_hash())
             .filter(|dna| !all_dnas.contains(dna));
 
-        // For any unrepresented DNAs, clean up those DNA-specific databases
-        for dna_hash in dnas_to_cleanup {
+        // For any unrepresented DNAs, purge data from those DNA-specific databases
+        for dna_hash in dnas_to_purge {
+            // Delete all data from authored databases
+            // TODO: we should delete the entire database directory, since these databases
```

Comment:
There is the `.path()` of a database could be used to delete the database file `std::fs::remove_file(db.path())`. What do you think about doing that?

Reply:
Agree, prefer removing the file. All the database files are in one directory, it's flat - so removing the file sounds like the best option to me

Reply:
I wonder if the connection pool needs to be cleaned up too then. I'll write a test if there isn't one already.